IMOS/imosParameters.txt

```diff
@@ -1,209 +1,207 @@
-WSPD,                1, wind_speed,                                                                               m s-1,         ,              ,                                  M, 999999.0, ,         ,         float
-WSPD_AVG,            1, wind_speed,                                                                               m s-1,         ,              ,                                  M, 999999.0, ,         ,         float
-WSPD_MIN,            1, wind_speed,                                                                               m s-1,         ,              ,                                  M, 999999.0, ,         ,         float
-WSPD_MAX,            1, wind_speed,                                                                               m s-1,         ,              ,                                  M, 999999.0, ,         ,         float
-WSSH,                0, sea_surface_wave_spectral_significant_height,                                             m,             ,              ,                                  W, 999999.0, 0.0,      100.0,    float
-WWAV,                1, sea_surface_wind_wave_to_direction,                                                       degree,        clockwise,     true north,                        W, 999999.0, 0.0,      360.0,    float
-WWDS,                0, sea_surface_wind_wave_directional_spread,                                                 degree,        clockwise,     true north,                        W, 999999.0, 0.0,      360.0,    float
-WWPD,                0, sea_surface_wind_wave_from_direction_at_variance_spectral_density_maximum,                degree,        clockwise,     true north,                        W, 999999.0, 0.0,      360.0,    float
-WWPD_MAG,            0, sea_surface_wind_wave_from_direction_at_variance_spectral_density_maximum,                degree,        clockwise,     magnetic north,                    W, 999999.0, 0.0,      360.0,    float
-WWPP,                0, sea_surface_wind_wave_period_at_variance_spectral_density_maximum,                        Second,        ,              ,                                  W, 999999.0, 0.0,      100.0,    float
-WWSH,                1, sea_surface_wind_wave_significant_height,                                                 m,             ,              ,                                  W, 999999.0, 0.0,      100.0,    float
-XCO2_AIR,            1, mole_fraction_of_carbon_dioxide_in_air,                                                   1e-6,          ,              ,                                  K, 999999.0, ,         ,         float
-XCO2_WATER,          0, mole_fraction_of_carbon_dioxide_in_sea_water,                                             1e-6,          ,              ,                                  K, 999999.0, ,         ,         float
\ No newline at end of file
+%
```

Comment:
diff shows all lines have changed. Did you change the line feed? It's impossible to see what happened there.